### ISSUE TITLE
Improve formatting of podcasts' description on podcast providers

### ISSUE THREAD

**Issue by aarangop** [Nonlinear-EA/The-Nonlinear-Library#40]
The old code generated a description which contained a link to the original post and a portion of the episode's written content below. The new code formats the text differently and doesn't look as good.

Old:
![image](https://github.com/Nonlinear-EA/The-Nonlinear-Library/assets/11898207/f03e3fbe-21b4-401d-b523-9a9dad1d3c9c)

New:
![image](https://github.com/Nonlinear-EA/The-Nonlinear-Library/assets/11898207/85c3163e-f0e2-45f2-b251-7ad6bb7d3fe8)

Modify the output of the new functions so the post description resembles the old code.

**Comment by maximumpeaches** [comment]
I think it would be important for us to include the link to the original description in our code, before we deploy the code for the Nonlinear Library podcast. Does that make sense to you too? I can imagine people complaining if we launch the new code and break that link.

I can't tell the difference in text formatting. The text looks very similar to me. Where are these screenshots from?

**Comment by aarangop** [comment]
Yeah I agree that we should include the link to the original article.

I renamed the issue because maybe 'formatting' wasn't the right word. I meant the structure of the description's html, which was something like:

Link to original article
<br/>
<br/>
Description.

Screenshot's from Spotify.
